db/migrate/20200727113800_change_va_caution_flag_date_type.rb

```diff
@@ -0,0 +1,16 @@
+class ChangeVaCautionFlagDateType < ActiveRecord::Migration[4.2]
+    def change
+        drop_table :va_caution_flags
```

Comment:
Seems like your goal is to change a data type - what's up with dropping the table first? I think it's possible to simply change the column data type directly.

Reply:
I can't recall what the error was, but there is a warning that blocks the migration when you try to  change the column data type. I figured since we are not yet using this table, I can just go ahead and drop it and recreate it.

Reply:
Was the message from `strong_migrations`- [link](https://github.com/ankane/strong_migrations#changing-the-type-of-a-column)?

I didn't realize the table was unused 👍 . Still, it seems unorthodox that a migration named "ChangeDataType" contains `drop_table` & `create_table`.  I'd recommend that you create 2 separate migrations files e.g. `XXX_drop_va_caution_flags.rb` + `XXX_create_va_caution_flags.rb`.